etc/gensbom/README.md

```diff
@@ -0,0 +1,114 @@
+# Generating SBOMs From Container Images Using Syft
+
+## Using the Container
+
+**Prerequisites:**
+
+* `podman`
+* [`config.json`](https://github.com/google/go-containerregistry/tree/main/pkg/authn#docker-config-auth)
+  in the current working directory
+* `TPA_SERVICE_URL`, holding the URL to the running TPA service, e.g.
+  `my.tpa.instance.abc:8765`
+* `TPA_AUTH_TOKEN`, holding the valid authorization token, e.g.
+  `Bearer XXXXXXXXXX`
+
+**Running:**
+
+```
+# Assume `images.txt` is a list of images (one image per line) in the current
+# working directory
```

Comment:
It would be helpful (to me, anyway) to provide an `example-images.txt` file in this directory, with public images from quay.io. Just so I can see the script working without having to authenticate to quay. Is that possible?